tests/cypress/e2e/filters-data.frame.cy.js

```diff
@@ -0,0 +1,150 @@
+describe("data.frame filters", () => {
+  beforeEach(() => {
+    cy.visit("http://127.0.0.1:5555/");
+    cy.waitForStabilityAndCatchError("body");
+    cy.get(".cy-active-summary-table tbody tr").as("activeSummary");
```

Comment:
I don't like the fact id is duplicated with `"cy_"` prefix. I propose following convention: since we need to be aware that certain element is tested let's just add `cy-testable` class to the existing class (as a second class). Then selector for this like will be
```suggestion
    cy.get(".cy-testable.teal_active_summary_filter_panel tbody tr").as("activeSummary");
```

Reply:
👍🏽 Let's try to find out if it's possible to implement the tests without any change to the classes in the package as we talked about. If unsuccessful, we can use this convention, I like it.

Reply:
I respect the idea of having a distinction between what's testable and what is not so app developer won't change this casually (I hope at least). It will be good to write down this convention somewhere that "testable" means don't touch it ;)

Also, I think we need to review all html classes to follow some "template". They seem to be random

Reply:
> Also, I think we need to review all html classes to follow some "template". They seem to be random

Agreed

Reply:
if `testable` is propagated in some good-practice guidelines then this should be ok